pymc/logprob/rewriting.py

```diff
@@ -376,6 +374,12 @@ def incsubtensor_rv_replace(fgraph, node):
 
 logprob_rewrites_db.register("measurable_ir_rewrites", measurable_ir_rewrites_db, "basic")
 
+# Split max_and_argmax
+# We only register this in the measurable IR db because max does not have a grad implemented
```

Comment:
What do you mean by "max doesn't have a grad implemented"? `pt.grad(pt.max(x).sum(), x)` works fine. Is it because internally that graph is using `MaxAndArgmax`, not `Max` itself?